scripts/provision-user.py

```diff
@@ -26,6 +28,10 @@
         {
             'Name': 'email_verified',
             'Value': 'True'
+        },
+        {
+            'Name': 'custom:tenantId',
+            'Value': 'tenant1'
```

Comment:
When provisioning to a non multi tenant solution will this cause any issues? I don't think so

Reply:
It won't. It'll be an unused attribute of the users